modules/videoio/src/cap_mjpeg_decoder.cpp

```diff
@@ -40,652 +40,11 @@
 //M*/
 
 #include "precomp.hpp"
-#include <deque>
-#include <stdint.h>
+#include "opencv2/core/private.cap_avi_container.hpp"
```

Comment:
Let's rename this header to `container_avi.private.hpp` and it should be moved to the _videoio_ module.